pphtml.py

```diff
@@ -357,17 +357,18 @@ def imageSummary(self):
 
     def coverImage(self):
         """
-        cover image width must be >=650 px and height must be >= 1000 px
+        cover image width should be >=1600 px and height should be >= 2560 px
+        https://www.pgdp.net/wiki/DP_Official_Documentation:PP_and_PPV/Post-Processing_FAQ#Cover_image
         """
         r = ["[pass] cover image dimensions check"]
         for t in self.filedata:
             u = t.split("|")
-            if u[0] == "cover.jpg":
+            if u[0] in ("cover.jpg", "cover.png"):
                 width, height = u[2].split("x")
-                if int(width) < 650 or int(height) < 1000:
+                if int(width) < 1600 or int(height) < 2560:
```

Comment:
Would it be better to accept landscape covers without complaint - I think this assumes portrait.
See recent discussion in https://github.com/DistributedProofreaders/guiguts-py/issues/960 and GG2 pphtml fix in https://github.com/DistributedProofreaders/guiguts-py/pull/969